src/openapi/streaming/streaming.ts

```diff
@@ -179,11 +181,17 @@ class Streaming extends MicroEmitter<EmittedEvents> {
         baseUrl: string,
         authProvider: AuthProvider,
         options?: Partial<types.StreamingConfigurableOptions>,
+        callbacks?: {
+            onProbe: (message: types.ProbeControlMessage) => void

```

Comment:
If future callbacks are added, we won't want to force the user to specify them all - if we make this optional too, it'll save having to consider it when a later committer comes along